object_survey/SurveyScan[] scans